client/src/utils/jobsUtil.js

```diff
@@ -93,7 +94,9 @@ export function getJobDisplayInformation(job, app) {
     // Improve any values with app information
     try {
       try {
-        display.systemName = getSystemName(app.exec_sys.host);
+        display.systemName = getSystemName(
+          getExecSystemFromId(app, job.execSystemId)?.host
+        );
```

Comment:
For some reason locally, I'm not seeing the selected system render on job details, in "Execution System":
![Screenshot 2024-03-19 at 12 48 49 PM](https://github.com/TACC/Core-Portal/assets/20326896/11c7b880-2f03-40e7-94cf-f98cc0791f8e)

Reply:
good catch, fixed it now. Somewhere in the commits in this PR, I broke this.
![Screenshot 2024-03-21 at 1 46 01 PM](https://github.com/TACC/Core-Portal/assets/2568355/babd6a15-de8a-4646-a492-0bc8438253c0)